diff --git a/node_modules/@opentelemetry/context-async-hooks/build/src/AsyncHooksContextManager.js b/node_modules/@opentelemetry/context-async-hooks/build/src/AsyncHooksContextManager.js
index a54e4fc..49c3ef3 100644
--- a/node_modules/@opentelemetry/context-async-hooks/build/src/AsyncHooksContextManager.js
+++ b/node_modules/@opentelemetry/context-async-hooks/build/src/AsyncHooksContextManager.js
@@ -17,7 +17,13 @@
 "use strict";
 exports.AsyncHooksContextManager = void 0;
 const api_1 = require("@opentelemetry/api");
-const asyncHooks = require("async_hooks");
+// Provide a shim for browsers
+const asyncHooks = typeof window === 'undefined' 
+  ? require("async_hooks") 
+  : { 
+      createHook: () => ({ enable: () => {}, disable: () => {} }),
+      executionAsyncId: () => 0
+    };
 const AbstractAsyncHooksContextManager_1 = require("./AbstractAsyncHooksContextManager");
 class AsyncHooksContextManager extends AbstractAsyncHooksContextManager_1.AbstractAsyncHooksContextManager {
     constructor() {